myproject/supermarket_manager/urls.py

```diff
@@ -25,4 +27,6 @@
     path("api/accounts", add_account, name="add-account"),
     path("api/accounts/<str:account_id>", get_account_detail, name="get-account-detail"),
     path("api/logins", login_account, name="login-account"),
+    path("api/accounts/updates/<str:account_id>", update_account, name="update-account"),
+    path("api/accounts/<str:account_id>/changes-password", change_password, name="change password")
```

Comment:
Should update the paths.
I think we no need `/updates/` and `/changes-password`.

Reply:
Done